### DISCUSSION TITLE
Issue moving files to mox

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1396] [Q&A] [answered]
I am trying to `wget` or `rsync` files to `scrubbed` dir

eg 
```
rsync -avz sr320@gannet.fish.washington.edu:/volume2/web/seashell/bu-mox/scrubbed/071521-cvBS/*sorted.bam .
```

error 
```
receiving incremental file list
12M_R1_val_1_bismark_bt2_pe.deduplicated.sorted.bam
rsync: write failed on "/gscratch/scrubbed/sr320/021022-CvBSnp/12M_R1_val_1_bismark_bt2_pe.deduplicated.sorted.bam": Disk quota exceeded (122)
rsync error: error in file IO (code 11) at receiver.c(393) [receiver=3.1.2]
rsync: [generator] write error: Broken pipe (32)
```

more
```
mmlsquota -j scrubbed gscratch
                         Block Limits                                    |     File Limits
Filesystem type             KB      quota      limit   in_doubt    grace |    files   quota    limit in_doubt    grace  Remarks
gscratch   FILESET  103462043264 94371840000 103809024000  346979072   6 days |  3901061 108000000 120000000     2001     none 
```

ideas?

**Comment by sr320** [comment, marked as answer]
Could have just been a glitch... seems to be working now, will leave up until it completes.

**Reply by kubu4** [reply]
Currently, `/gscratch/scrubbed/` _is_ over quota...

![image](https://user-images.githubusercontent.com/4514104/153471270-31d6c07e-afd0-4329-95fb-9edac4dda9fa.png)


So, who knows what will happen. I've made a post in the RC Slack about it.

**Reply by kubu4** [reply]
Dang, there are two users who (when combined) are consuming >50% of all of the space on `/gscratch/scrubbed`. Wild

**Reply by kubu4** [reply]
FYI, Mox UW-IT is aware of this and have a plan in place to start removing files, based on age (probably starting at files > 21 days old).
